mavros_extras/src/plugins/trajectory.cpp

```diff
@@ -172,50 +167,74 @@ class TrajectoryPlugin : public plugin::PluginBase {
 			trajectory.point_5[6] = acceleration_point_5.x();
 			trajectory.point_5[7] = acceleration_point_5.y();
 			trajectory.point_5[8] = acceleration_point_5.z();
-			double yaw_ned_point_5 = wrap_pi(-req->point_5.yaw + (M_PI / 2.0f));
-			trajectory.point_5[9] = yaw_ned_point_5;
+			trajectory.point_5[9] = wrap_pi(-req->point_5.yaw + (M_PI / 2.0f));
 			trajectory.point_5[10] = req->point_5.yaw_rate;
 
-			// [[[end]]] (checksum: c19ffbd7dee05e9242b465c7642c3656)
+			// [[[end]]] (checksum: de8ef970a43d4b07fb7581c856c95980)
 		} else {
 			//[[[cog:
-			//for p in "12345":
-			//     cog.outl("trajectory.point_{p}[3] = req->point_{p}.velocity.x;".format(**locals()))
-			//     cog.outl("double yaw_ned_point_{p} = wrap_pi(-req->point_{p}.yaw + (M_PI / 2.0f)); \n".format(**locals()))
-			//     cog.outl("trajectory.point_{p}[4] = yaw_ned_point_{p};".format(**locals()))
-			//     cog.outl("trajectory.point_{p}[5] = req->point_{p}.yaw_rate; \n".format(**locals()))
+			//for p, index in zip("12345", "01234"):
+			//     cog.outl("trajectory.point_{p}[3] = req->time_horizon[{index}];".format(**locals()))
```

Comment:
Note: follow python style, 4 spaces. Use one space between comment slashes and code.